test/spec/modules/mobianRtdProvider_spec.js

```diff
@@ -23,70 +23,63 @@ describe('Mobian RTD Submodule', function () {
     ajaxStub.restore();
   });
 
-  it('should return no_risk when server responds with garm_no_risk', function () {
+  it('should return risk level when server responds with garm_risk', function () {
```

Comment:
the diff on the tests looks huge, but isn't -- we're just removing the testing of the old `garm_x_risk` since we always expect `garm_risk` coming in.